src/components/scenes/TransactionsExportScene.tsx

```diff
@@ -385,6 +386,7 @@ class TransactionsExportSceneComponent extends React.PureComponent<Props, State>
     }
 
     await Share.open({
+      failOnCancel: false,
```

Comment:
Shouldn't we use `showError`?

Reply:
No. This prevents an error when the user simply cancelled sharing. Not an error at all.

Reply:
I mean shouldn't we use `showError` instead of `console.log(error)` in the catch?

Reply:
ah ok. got it. will change